app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -273,6 +272,7 @@ def get_parent_host(headers)
       hostnames = headers["HTTP_X_FORWARDED_HOST"]
       host = hostnames.split(/[,,:]/)[0].strip if hostnames
       host || URI.parse(Setting[:foreman_url]).host
+      "loadbalancer.example.com"
```

Comment:
I assume this is for testing and will get cleaned up.

Reply:
Yeah, this is one of the two "hard codings" I refer to in the description :)